jerry-core/include/jerryscript-core.h

```diff
@@ -308,6 +308,19 @@ typedef struct
  */
 typedef struct jerry_context_t jerry_context_t;
 
+/**
+  * Enum that contains the supported binary operation types
```

Comment:
Nitpick: the `*` is not in the correctly column.

Reply:
Good catch, fixed it.